atom/common/crash_reporter/crash_reporter_linux.h

```diff
@@ -54,6 +56,7 @@ class CrashReporterLinux : public CrashReporter {
   uint64_t process_start_time_;
   pid_t pid_;
   std::string upload_url_;
+  bool upload_to_server_;
```

Comment:
Probably should be initialized to `true` in the `CrashReporterLinux` constructor for consistency.